pysb/tests/test_kappa.py

```diff
@@ -183,13 +198,28 @@ def test_influence_map_kasa():
 def test_unicode_strs():
     Monomer(u'A', [u'b'], {u'b':[u'y', u'n']})
     Monomer(u'B')
-    Rule(u'rule1', A(b=u'y') >> B(), Parameter(u'k', 1))
+    Rule(u'rule1', A(b=u'y') + None >> None + B(),
```

Comment:
This change should be reverted so it's back to just testing the unicode thing.

Reply:
The `None`s are now required with Kappa 4 for transforming one agent into another, otherwise the error `Left hand side/right hand side agent mismatch` appears. I changed the reaction pattern in this test to a state change to avoid this.

Reply:
Of course, good fix.